src/plugins/intel_cpu/src/nodes/subgraph.cpp

```diff
@@ -965,11 +992,34 @@ void Subgraph::SubgraphExecutor::parallel_forNd(const std::function<void(jit_sni
                 tmp /= dom[j];
             }
 
-            caller(call_args, indexes.data());
+            caller(call_args, indexes);
         }
     });
 }
 
+void Subgraph::SubgraphExecutor::execute(dnnl::stream strm, std::vector<MemoryPtr>& inMemPtrs, std::vector<MemoryPtr>& outMemPtrs) {
+    if (m_in_requested_descs.empty())
+        exec_impl(inMemPtrs, outMemPtrs);
+    else
+        reorder_execute(strm, inMemPtrs, outMemPtrs);
+}
+
+void Subgraph::SubgraphExecutor::reorder_execute(dnnl::stream strm, std::vector<MemoryPtr> inMemPtrs, const std::vector<MemoryPtr>& outMemPtrs) {
+    const auto internal_buffer_size = static_cast<size_t>(m_nthreads) * m_buffer_scratchpad_size;
```

Comment:
Maybe we should this value as class field? Then we don't need to calculate it twice